container_api/src/config.rs

```diff
@@ -139,14 +139,15 @@ impl TryFrom<DNAConfiguration> for Dna {
 }
 
 /// An instance combines a DNA with an agent.
-/// Each instance has its own storage and logger configuration.
+/// Each instance has its own network, storage and logger configuration.
 #[derive(Deserialize, Serialize, Clone)]
 pub struct InstanceConfiguration {
     pub id: String,
     pub dna: String,
     pub agent: String,
     pub logger: LoggerConfiguration,
     pub storage: StorageConfiguration,
+    pub network: String,
```

Comment:
I would make it optional (`Option<String>`) because I don't think this should be in the config files as the container user shouldn't (have to) even know that we have separated out our network module from core (which is the main reason for having this config).

Reply:
just pushed Option<String> fix